molecule/command/destroy.py

```diff
@@ -48,6 +48,7 @@ def execute(self, exit=True):
             return e.returncode, e.message
         self.molecule.remove_templates()
         self.molecule.remove_inventory_file()
+        self.molecule.remove_vars_files()
```

Comment:
We should assert this is called in [test_destroy](https://github.com/metacloud/molecule/blob/master/test/unit/command/test_destroy.py).